package.json

```diff
@@ -77,4 +77,4 @@
     "_comment": "Replace 'reactTypescript' with 'react' if you want use React without Typescript",
     "projectType": "reactTypescript"
   }
-}
\ No newline at end of file
+}
```

Comment:
Skip unnecessary files in your commit

Reply:
You are totally right! Thanks for your review.